src/test/java/seedu/address/logic/commands/LogCommandTest.java

```diff
@@ -103,12 +120,67 @@ public void equals() {
         // different types -> returns false
         assertFalse(standardCommand.equals(new ClearCommand()));
 
-        // different index -> returns false
+        // different name -> returns false
         assertFalse(standardCommand.equals(new LogCommand(secondStudySpotInTypicalStudySpots, copyStudiedHours,
                 false, false, false)));
 
         // different descriptor -> returns false
         assertFalse(standardCommand.equals(new LogCommand(secondStudySpotInTypicalStudySpots, differentStudiedHours,
                 false, false, false)));
+
+        // same name but different studied hours -> returns false
+        assertFalse(standardCommand.equals(new LogCommand(firstStudySpotInTypicalStudySpots, differentStudiedHours,
+                false, false, false)));
+
+        // same name same hours, different reset -> returns false
+        assertFalse(standardCommand.equals(new LogCommand(firstStudySpotInTypicalStudySpots, studiedHours,
+                true, false, false)));
+
+        // same name same hours, different override -> returns false
+        assertFalse(standardCommand.equals(new LogCommand(firstStudySpotInTypicalStudySpots, studiedHours,
+                false, true, false)));
+
+        // same name same hours, different reset all -> returns false
+        assertFalse(standardCommand.equals(new LogCommand(firstStudySpotInTypicalStudySpots, studiedHours,
+                false, false, true)));
+
+        LogCommand resetAllCommand = new LogCommand(null, null, false, false, true);
+        LogCommand sameResetAllCommand = new LogCommand(null, null, false, false, true);
+
+        // name == null, studied hours == null, different commands
+        assertFalse(standardCommand.equals(resetAllCommand));
+
+        // name == null, studied hours == null, same command
+        assertTrue(resetAllCommand.equals(sameResetAllCommand));
+
```

Comment:
Can do the ` -> returns true ` / ` -> returns false ` thing so it's helpful for future devs